plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClientModule.java

```diff
@@ -56,6 +58,8 @@ public void setup(Binder binder)
     @ForBaseJdbc
     public ConnectionFactory getConnectionFactory(BaseJdbcConfig config, CredentialProvider credentialProvider)
     {
-        return new DriverConnectionFactory(new Driver(), config, credentialProvider);
+        Properties connectionProperties = new Properties();
+        connectionProperties.put("reWriteBatchedInserts", "true");
```

Comment:
Can you move this change to a separate PR, like you did in https://github.com/trinodb/trino/pull/12411 ?

Reply:
Sure.